environs/config/config.go

```diff
@@ -634,6 +642,20 @@ func Validate(cfg, old *Config) error {
 		}
 	}
 
+	if raw, ok := cfg.defined[ContainerInheritProperiesKey].(string); ok && raw != "" {
+		rawProperties := strings.Split(raw, ",")
+		propertySet := set.NewStrings()
+		for _, prop := range rawProperties {
+			propertySet.Add(strings.TrimSpace(prop))
+		}
+		whiteListSet := set.NewStrings("apt-primary", "apt-sources", "apt-security", "ca-certs")
```

Comment:
Let's define the supported values as a slice outside this func and use that to init the set

Reply:
done